### PR TITLE
Initial Caffe Importer

### PR DESCRIPTION
**Pull request by gcucurull**
It still doesn't load the caffemodel to our custom format, but it is a prof of concept of how to open a caffemodel and get the information and parameters for each layer.

### CODE REVIEW

**Comment by edgarriba** [comment]
@gcucurull The tests are failing because we use CaffeImporter as dummy case. Could you switch them off? Ideally we should create a DummyCaffeImporter for testing

**Comment by gcucurull** [comment]
Ok, I'll change it later, I didn't notice it when I sent the PR.

**Comment by gcucurull** [comment]
I have added the DummyImporter, but the tests are still failing, because there is no DummyExporter I assume.
I feel like this is outside the scope of this PR, the tests should not be using CaffeImporter until we really want to test it, the same way as we are not still testing Lasagne or Matconvnet.

File: modules/importers/DummyCaffeImporter.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from core.BaseImporter import BaseImporter
```
Comment (edgarriba) [inline modules/importers/DummyCaffeImporter.py:1]: @gcucurull maybe we can place it inside tests/importers ?

**Comment by gcucurull** [comment]
I have "disabled" the testing in a dirty way. I know it is not the right thing to do, but it is outside the scope of this PR, so I think this PR should be merged and then fix the tests in another PR.

**Comment by edgarriba** [comment]
@gcucurull Is it ready to merge?

**Comment by gcucurull** [comment]
I guess it is, it's not the final version but it is a basic Caffe Importer to extend

**Comment by edgarriba** [comment]
oki! I'll merge it

**Pull request merged**